examples/sample-key

```diff
@@ -0,0 +1 @@
+0000000000000000000000000000000000000000000000000000000000000003
```

Comment:
any meaning for this key ? curiosity

Reply:
nah, 00 doesnt work because it's a point in infinity, so the keys are 01, 02 and 03 lol. If you see the test, there are 3 different keys